src/Illuminate/Auth/Middleware/EnsureEmailIsVerified.php

```diff
@@ -4,28 +4,73 @@
 
 use Closure;
 use Illuminate\Contracts\Auth\MustVerifyEmail;
-use Illuminate\Support\Facades\Redirect;
+use Illuminate\Contracts\Routing\ResponseFactory;
+use Illuminate\Routing\UrlGenerator;
```

Comment:
The import should be changed to `Illuminate\Contracts\Routing\UrlGenerator` as the docblock below correctly states.

Reply:
Updated!